.gitignore

```diff
@@ -1,4 +1,26 @@
 
 .env
 .idea
-venv
\ No newline at end of file
+venv
+
+app/routers/__pycache__/user.cpython-312.pyc
```

Comment:
It should be handled by the following line `**/__pycache__/` instead all the explicit mentioning